custom_model_runner/CHANGELOG.md

```diff
@@ -4,7 +4,7 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-#### [Current]
+#### [1.9.9] - 2022-09-29
 ##### Changed
 - bump com.datarobot.datarobot-prediction package to 2.2.1
```

Comment:
add a sentence that it fixed the dependency

Reply:
DONE